lib/TabsNew/TabsDragLine.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { string } from 'prop-types';
+import cx from 'classnames';
+
+export function TabsDragLine({ side }) {
+  const className = cx('tab-drag-line absolute top-0 h-full z-10', {
+    [side]: true
+  });
+
+  return <span className={className} />;
+}
+
+TabsDragLine.propTypes = {
```

Comment:
💚 You could also attach the side options to the component, like
```javascript
TabsDragLine.sides = {
  left: 'left',
  right: 'right',
  whole: 'whole',
}
```
either that, or use `oneOf(['left', 'right', 'whole'])` in the prop types. Currently we have to go in to the styles to know what our options are (or the story to be fair!)

Very 💚 💚 💚 comment!

Reply:
I'm into the first suggestion, it's like a built in const